src/Symfony/Bundle/FrameworkBundle/Test/WebTestAssertions.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use PHPUnit\Framework\Assert;
+use Symfony\Bundle\FrameworkBundle\Client;
+
+/**
+ * Ideas borrowed from Laravel Dusk's assertions.
+ *
+ * @see https://laravel.com/docs/5.7/dusk#available-assertions
+ */
+trait WebTestAssertions
+{
+    /** @var Client|null */
+    protected static $client;
+
+    public static function assertResponseIsSuccessful(): void
```

Comment:
Do these methods really need `public` visibility, or will `protected` suffice?

Reply:
I prefer to keep them public like the `Assert` class does, in case someone wants to test it or in case someone doesn't want to use the trait.

Reply:
But how would you be able to test the trait _without_ using it?

Reply:
Precisely by making its methods static and public 😄 
I just copied the behavior of the `Assert` class, that's all, and I think it's fine as-is

Reply:
> Precisely by making its methods static and public 😄

Can you give a concrete example?

I don't think there is a need to unnecessarily use a visibility that is not needed.

Reply:
I have no special argument, I just think it's fine to be able to use it without using the trait